akd/src/proto/audit.rs

```diff
@@ -0,0 +1,732 @@
+// This file is generated by rust-protobuf 2.8.1. Do not edit
+// @generated
+
+// https://github.com/Manishearth/rust-clippy/issues/702
```

Comment:
This looks like a broken link. Also, any reason we're using this fork instead of [rust-lang/rust-clippy](https://github.com/rust-lang/rust-clippy)?